var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -43,6 +43,10 @@ class Charmpp(Package):
     # Patch is no longer needed in versions 6.8.0+
     patch("mpi.patch", when="@:6.7.1")
 
+    # Patch for AOCC
+    patch('charm_6.7.1_aocc.patch', when="@6.7.1 %aocc", level=1)
+    patch('charm_6.8.2_aocc.patch', when="@6.8.2 %aocc", level=3)
```

Comment:
Are you patching only these two versions because they are the only one tested with `%aocc` or because they are the only one that need fixing?

Reply:
keeping NAMD 2.12 and 2.13 in mind we had changed only for these two versions and tested with only for these versions with `%aocc`

Reply:
Would `namd@2.14 %aocc` need patching too or does it work out of the box?

Reply:
`namd@2.14 %aocc` doesn't required patching as it is working out of the box